src/devtools/client/debugger/src/components/SecondaryPanes/Frames/NewFrames.tsx

```diff
@@ -44,12 +46,9 @@ function FramesRenderer({
   const replayClient = useContext(ReplayClientContext);
   const sourcesState = useAppSelector(state => state.sources);
   const { rangeForSuspense: focusWindow } = useContext(FocusContext);
+  const { endpoint } = useContext(SessionContext);
   const dispatch = useAppDispatch();
 
-  if (focusWindow === null) {
-    return null;
-  }
```

Comment:
I think this check was in place just to make TypeScript happy, but the `getAsyncParentPauseIdSuspense` function _already_ had a check for `focusWindow === null` so I just updated its type to accept a null param.

(If this ever _did_ return `null`, it hid the stack entirely which isn't what we'd want anyway.)